R/RunDiagnostics.R

```diff
@@ -83,6 +99,8 @@ runCohortDiagnostics <- function(packageName = NULL,
                                  runCohortOverlap = TRUE,
                                  runCohortCharacterization = TRUE,
                                  covariateSettings = FeatureExtraction::createDefaultCovariateSettings(),
+                                 runTemporalCohortCharacterization = FALSE,
+                                 temporalCovariateSettings = NULL,
```

Comment:
As I mentioned before, I would here just assign the covariate settings you describe in the documentation. That makes it more transparent what happens.

Reply:
Sorry. i think i am not following, and I know you are right..  
To make it transparent, i had added this to the documentation
https://github.com/OHDSI/CohortDiagnostics/blob/ec4cf215144efe41a3de52bac7d3ae2e5228570d/R/RunDiagnostics.R#L26

but i think you are saying, there is a better way

Reply:
Yes, why not assign the default value, instead of making it so a NULL value mysteriously leads to the default value? So change line 103 to:
```r
temporalCovariateSettings = FeatureExtraction::createTemporalCovariateSettings(useConditionOccurrence = TRUE,
                                                                                 useConditionEraStart = TRUE,
                                                                                 useDrugEraStart = TRUE,
                                                                                 useProcedureOccurrence = TRUE,
                                                                                 useMeasurement = TRUE,
                                                                                 useObservation = TRUE,
                                                                                 temporalStartDays = c(-365,-30,0,1,31),
                                                                                 temporalEndDays = c(-31,-1,0,30,365)),
```

Reply:
Got it! thanks for your guidance!